node/options_test.go

```diff
@@ -1425,3 +1425,40 @@ func TestWithPeerSignatureHandler_OkPeerSignatureHandlerShouldWork(t *testing.T)
 	assert.Equal(t, peerSigHandler, node.peerSigHandler)
 	assert.Nil(t, err)
 }
+
+func TestWithPeerSignatureHandler_EnableSignTxWithHashEpochShouldWork(t *testing.T) {
```

Comment:
change tests names from `TestWithPeerSignatureHandler` to `TestWithEnableSignTxWithHashEpoch`

Reply:
done.